File: leetcode/1006.go
package leetcode

func clumsy(N int) int {
	ans := []int{0, 1, 2, 6, 7}
	if N <= 4 {
		return ans[N]
	}
	if N%4 == 0 {
		return N + 1
	} else if N%4 <= 2 {
		return N + 2
	} else {
		return N - 1
	}
	return -1
}

File: leetcode/1006.cpp
class Solution {
public:
    int clumsy(int n) {
        int ans[5] = {0,1,2,6,7};
        if (n <= 4) return ans[n];
        if (n%4 == 0)
            return n + 1;
        else if (n%4 <= 2)
            return  n + 2;
        else return n - 1;
    }
};